airbyte-webapp/src/components/base/Button/types.tsx

```diff
@@ -2,6 +2,7 @@ export interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElemen
   full?: boolean;
   danger?: boolean;
   secondary?: boolean;
+  light?: boolean;
```

Comment:
I don't think we should add a new styling here for the button. We have a design library for how the button should look like. But leaving @Upmitt to commit to this from the design perspective.